### PR TITLE
chore: formatting & copy updates

### PR DESCRIPTION
**Pull request by fabe**
This updates the formatting and copy. I also `tildify` the paths:

image
image

### CODE REVIEW

**Comment by Jwhiles** [comment]
are we publishing this under the contentful org? If so, I think we should also update the commands to `npx @contentful/create-contentful-app ...`

File: lib/login.js [hunk 1]
```diff
@@ -33,7 +34,7 @@ module.exports = async function login() {
 
     return mgmtToken;
   } catch (err) {
-    console.log('Failed to login into Contentful');
+    console.log(chalk.red('Failed to login into Contentful.'));
```
Comment (kdamball) [inline lib/login.js:37]: Is red going to look in dark editors?

Comment (fabe) [reply lib/login.js:37]: It's using the `red` of the theme, so it'll be readable. Chalk handles everything.

Comment (Jwhiles) [inline lib/login.js:37]: I'd prefer if we could change this to something like

```suggestion
    console.log(`${chalk.red('error:')} Failed to login into Contentful.'));
```
Output printed entirely in red is very hard to see on terminals with dark backgrounds

**Comment by fabe** [comment]
> are we publishing this under the contentful org? If so, I think we should also update the commands to `npx @contentful/create-contentful-app ...`

Yes, we should update the readme once we have the package without `@contentful`!

File: lib/prompt-app-definition.js [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
   const appDefinitionSettings = await inquirer.prompt([
     {
       name: 'name',
       message: `App name (${path.basename(process.cwd())}):`
     },
     {
       name: 'locations',
-      message:
-        'Select where your app can be rendered. \n NOTE: Read more about app locations at https://ctfl.io/app-locations\n',
+      message: `Select where your app can be rendered in:`,
```
Comment (Jwhiles) [inline lib/prompt-app-definition.js:21]: "can be rendered in" sounds  strange to me. I think "can be rendered" would be fine.

File: lib/login.js (hunk 1 above)
Comment (fabe) [reply lib/login.js:37]: Good point, I'll do that for all error messages.

File: lib/index.js [hunk 3]
```diff
@@ -2,42 +2,61 @@
...
-  console.log(`   ${mainCommand} create-definition`);
-  console.log();
+  console.log(`
+${chalk.cyan('Success!')} Created a new Contentful app in ${chalk.bold(
+    tildify(path.resolve(process.cwd(), folder))
+  )}.
+
+We suggest that you begin by running:
+
+    ${chalk.cyan(`cd ${folder}`)}
```
Comment (kdamball) [inline lib/index.js:31]: Same thing here with the colours. Can we make sure it doesn't conflict? I think the `dim` stuff you used somewhere might be the way to go :)

Comment (fabe) [reply lib/index.js:31]: It's using whatever `cyan` color is specified for the current theme, so the actual color will differ between themes. Our `inquirer.prompt` is also using `chalk.cyan` behind the scenes.

**Pull request merged**